lib/apollo-federation/schema.rb

```diff
@@ -64,7 +64,7 @@ def federation_2_prefix
 
         <<~SCHEMA
           extend schema
-            @link(url: "https://specs.apollo.dev/federation/v2.3"#{federation_namespace}, import: [#{(IMPORTED_DIRECTIVES.map { |directive| "\"@#{directive}\"" }).join(', ')}])
+            @link(url: "https://specs.apollo.dev/federation/v2.3"#{federation_namespace}, import: [#{(get_imported_directives.map { |directive| "\"@#{directive}\"" }).join(', ')}])
```

Comment:
now you need to use `self.class.imported_directives` since its `class_attribute`